pages/reference/cli.md

```diff
@@ -1478,6 +1390,8 @@ Examples:
 	$ balena push 10.0.0.1
 	$ balena push 10.0.0.1 --source <source directory>
 	$ balena push 10.0.0.1 --service my-service
+	$ balena push 10.0.0.1 --env MY_ENV_VAR=value --env my-service:SERVICE_VAR=value
+	$ balena push 10.0.0.1 --nolive
```

Comment:
Most of the updates (but not all) that this PR brings to `cli.md` were already reflected on the website at:
https://www.balena.io/docs/reference/cli/

So clearly there something else that brings updates to the website, outside of the PR-merging process. It would be nice to understand better how that works... Still, for example the `--nolive` option above was not yet reflected on the website.

Reply:
I too find this out-of-band update to be confusing, +1 on clarity!

Reply:
Very shortly after the PR was merged to the `master` branch, the website was updated. So even though there's some other route for updating it, getting PRs merged also triggers an update.